### PR TITLE
[Enhance](DOE)Support parse default es iso datetime string

### PR DESCRIPTION
**Pull request by SaintBacchus**
# Proposed changes

Issue Number: close #xxx

## Problem summary
Using cctz to parse default es iso datetime string;

## Checklist(Required)

* [ ] Does it affect the original behavior
* [x] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by SaintBacchus** [comment]
run buildall

File: docker/thirdparties/custom_settings.env [hunk 1]
```diff
@@ -20,4 +20,4 @@
 # Do not use "_" or other sepcial characters, only number and alphabeta.
 # eg: CONTAINER_UID="doris-jack-"
 # NOTICE: change this uid will modify the file in docker-compose.
-CONTAINER_UID="doris--"
+CONTAINER_UID="doris-a-"
```
Comment (stalary) [inline docker/thirdparties/custom_settings.env:23]: Why modify it

File: fe/fe-core/src/main/java/org/apache/doris/external/elasticsearch/EsUtil.java [hunk 2]
```diff
@@ -108,22 +108,26 @@ public static ObjectNode getRootSchema(ObjectNode mappings, String mappingType)
...
             // remove dynamic templates, for ES 7.x and 8.x
-            checkNonPropertiesFields(mappings);
-            String firstType = mappings.fieldNames().next();
-            if (!"properties".equals(firstType)) {
-                // If type is not passed in takes the first type.
-                ObjectNode firstData = (ObjectNode) mappings.get(firstType);
-                // check for ES 6.x and before
-                checkNonPropertiesFields(firstData);
-                return firstData;
+            // checkNonPropertiesFields(mappings);
```
Comment (stalary) [inline fe/fe-core/src/main/java/org/apache/doris/external/elasticsearch/EsUtil.java:111]: Can rebase master

File: regression-test/conf/regression-conf.groovy [hunk 3]
```diff
@@ -86,7 +86,7 @@ hdfs_port=8120
 
 // elasticsearch catalog test config
 // See `docker/thirdparties/start-thirdparties-docker.sh`
-enableEsTest=false
+enableEsTest=true
```
Comment (stalary) [inline regression-test/conf/regression-conf.groovy:89]: CI not enable it.

**Review by stalary — APPROVED** [review]
LGTM

**Comment by SaintBacchus** [comment]
run p0

**Comment by SaintBacchus** [comment]
run p0

**Comment by SaintBacchus** [comment]
run p0

**Comment by stalary** [comment]
run p0

**Pull request merged**